hack/build-base-images.sh

```diff
@@ -8,7 +8,15 @@ source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 # determine the correct tag prefix
 tag_prefix="${OS_IMAGE_PREFIX:-"openshift/origin"}"
 
+os::util::ensure::gopath_binary_exists imagebuilder
+
+# image builds require RPMs to have been built
+os::build::release::check_for_rpms
```

Comment:
I feel like the moment we do this the "base, release, images" separation doesn't make sense.  It's really "release, images".  Base is only supposed to be images that don't depend on build artifacts.

Would recommend moving this code out of here and into build-images (building images/source).  Put the check for rpms there (and remove the other hacks).